server/game/cards/11-TCaR/FriendsInHighPlaces.js

```diff
@@ -0,0 +1,62 @@
+const ActionCard = require('../../actioncard.js');
+
+class FriendsInHighPlaces extends ActionCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onDudeCalledOut: event => 
+                    event.callee.controller === this.controller &&
+                    event.callee.influence > event.caller.influence &&
+                    !event.callee.booted &&
+                    event.canReject === true
+            },
+            message: context => {
+                this.game.addMessage('{0} uses {1} to refuse the callout without moving home booted', context.player, this);
+            },
+            handler: context => {
+                this.lastingEffect(ability => ({
+                    until: {
+                        onCardCallOutFinished: () => true
+                    },
+                    match: context.event.callee,
+                    effect: ability.effects.canRefuseWithoutGoingHomeBooted()
+                }));
+            }
+        });
+
+        this.action({
+            title: 'Shootout: Friends in High Places',
+            playType: 'shootout',
+            ifCondition: () => this.getPosseInfluence(this.controller) > this.getPosseInfluence(this.controller.getOpponent()),
+            ifFailMessage: context => {
+                this.game.addMessage('{0} uses {1} but does not make anyone a stud because their posse\'s total influence is less than the opposing posse\'s', context.player, this);
+            },
+            handler: context => {
+                this.game.promptForSelect(context.player, {
+                    activeTitlePrompt: 'Select a dude to make a stud',
+                    waitingTitlePrompt: 'Waiting for opponent to select a dude',
```

Comment:
```suggestion
                    activePromptTitle: 'Select a dude to make a stud',
                    waitingPromptTitle: 'Waiting for opponent to select a dude',
```
Typo in prompt property names.